### PR TITLE
doc:update acrn-shell.rst

### PR DESCRIPTION
**Pull request by mingqiangchi**
update this doc related with dump host & guest memory.

Signed-off-by: Mingqiang Chi <mingqiang.chi@intel.com>

### CODE REVIEW

File: doc/user-guides/acrn-shell.rst [hunk 1]
```diff
@@ -21,9 +21,12 @@ The ACRN hypervisor shell supports the following commands:
      - List all vCPUs in all VMs
    * - vcpu_dumpreg <vm_id> <vcpu_id>
      - Dump registers for a specific vCPU
-   * - dumpmem <hva> <length>
-     - Dump host memory, starting at a given address, and for a given length
-       (in bytes)
+   * - dump_host_mem <hva> <length>
```
Comment (deb-intel) [inline doc/user-guides/acrn-shell.rst]: Do you also need to add an entry for dump_guest_mem to the table as well?

Comment (mingqiangchi) [reply doc/user-guides/acrn-shell.rst]: i have added this entry to the table like this,  i am not sure if it is right?

+   * - dump_host_mem <hva> <length>
+     - Dump host memory, starting at a given address(hexadecimal), and for a given length
+       (decimal in bytes)
+   * - dump_guest_mem <vm_id> <gva> <length>
+     - Dump guest memory, vm id(decimal),starting at a given address(hexadecimal), and for a given length
+       (decimal in bytes)

File: doc/user-guides/acrn-shell.rst [hunk 2]
```diff
@@ -21,9 +21,12 @@ The ACRN hypervisor shell supports the following commands:
...
+     - Dump host memory, starting at a given address(hexadecimal), and for a given length
```
Comment (gvancuts) [inline doc/user-guides/acrn-shell.rst]: How about something like: Dump the host memory region as specified by the start of the region ``hva`` (in hexadecimal) and its length ``length`` (in bytes, decimal number).

File: doc/user-guides/acrn-shell.rst [hunk 3]
```diff
@@ -21,9 +21,12 @@ The ACRN hypervisor shell supports the following commands:
...
+       (decimal in bytes)
+   * - dump_guest_mem <vm_id> <gva> <length>
+     - Dump guest memory, vm id(decimal),starting at a given address(hexadecimal), and for a given length
```
Comment (gvancuts) [inline doc/user-guides/acrn-shell.rst]: Dump a User VM (guest) memory region based on the VM ID (``vm_id``, in decimal), the start of the memory region ``gva`` (in hexadecimal) and its length ``length`` (in bytes, decimal number).

File: doc/user-guides/acrn-shell.rst [hunk 4]
```diff
@@ -104,10 +107,10 @@ function ``acpi_idle_do_entry``.
 
    system map information
 
-dumpmem
-=======
+dump_host_mem
+=============
 
-``dumpmem mem_address`` provides the specified memory target data such as
+``dump_host_mem mem_address`` provides the specified memory target data such as
```
Comment (gvancuts) [inline doc/user-guides/acrn-shell.rst]: What is this ``mem_address`` parameter?

File: doc/user-guides/acrn-shell.rst [hunk 5]
```diff
@@ -129,6 +132,26 @@ pCPU number is 0x0000000000000004.
 
    acrn map information
 
+dump_guest_mem
+==============
+
+The ``dump_guest_mem`` command can dump guest memory according to the given
+vm id and guest virtual address.
```
Comment (gvancuts) [inline doc/user-guides/acrn-shell.rst]:
```suggestion
VM ID and guest virtual address (``gva``).
```

File: doc/user-guides/acrn-shell.rst (hunk 2 above)
Comment (mingqiangchi) [reply doc/user-guides/acrn-shell.rst]: ok will update

File: doc/user-guides/acrn-shell.rst (hunk 3 above)
Comment (mingqiangchi) [reply doc/user-guides/acrn-shell.rst]: will update

File: doc/user-guides/acrn-shell.rst (hunk 4 above)
Comment (mingqiangchi) [reply doc/user-guides/acrn-shell.rst]: will change it to hva

File: doc/user-guides/acrn-shell.rst (hunk 5 above)
Comment (mingqiangchi) [reply doc/user-guides/acrn-shell.rst]: will update, thanks

**Comment by mingqiangchi** [comment]
@gvancuts have updated, could you help review?

File: doc/user-guides/acrn-shell.rst [hunk 6]
```diff
@@ -104,10 +107,10 @@ function ``acpi_idle_do_entry``.
 
    system map information
 
-dumpmem
-=======
+dump_host_mem
+=============
 
-``dumpmem mem_address`` provides the specified memory target data such as
+``dump_host_mem hva,length`` provides the specified memory target data such as
```
Comment (gvancuts) [inline doc/user-guides/acrn-shell.rst]: Do you need to comma in between `hva` and `length`? If not, please remove it

Comment (mingqiangchi) [reply doc/user-guides/acrn-shell.rst]: have removed it, thanks

**Pull request merged**